src/app/app.component.ts

```diff
@@ -83,3 +81,53 @@ export class AppComponent extends withUnsubscribe() implements OnInit {
       });
   }
 }
+
+@Injectable()
+export class PageTitleStrategy extends TitleStrategy {
+  public constructor(
+    private title: Title,
+    private config: ConfigService,
+  ) { super(); }
+
+  /**
+   * Recursively iterates through the nested components to construct a title
+   * This is needed for components that are mounted in multiple locations
+   *
+   * @param componentState The component as defined in the route data
+   * @param routeState a `RouterStateSnapshot` representing the router state. This value should not change throughout calls.
+   * @returns A `string` representing the breadcrumb steps that the user took
+   */
+  private constructBreadcrumbTitle(componentState, routeState: RouterStateSnapshot): string {
+    const componentPageInfos = getPageInfos(componentState?.route.pageComponent);
+
+    // in the edge case where a page doesn't have page info, or a title stop searching through
+    // after it comes across a component that does not have a title and use the highest level of actitlescuracy possible
+    if (!componentPageInfos || !componentPageInfos[0]?.title) {
+      return "";
+    }
+
+    const componentPageInfo = componentPageInfos[0];
+
+    // the Angular title case pipe ensures that all titles start with a capital letter after each space
+    const componentTitle = componentPageInfo.title(routeState);
```

Comment:
100%. I have found where this will cause problems.

In the case where there are multiple mounting points for a page (e.g. Sites / Points), multiple `PageInfo` objects are returned.

I'll be fixing this and adding some tests to assert that it works as expected